### PR TITLE
Switch meson to using pip and update to 1.6.0

### PR DESCRIPTION
**Pull request by satmandu**
- Also switch `itstool` to pip...

##
Builds:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`
##
Tested & Working properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 
##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=updates crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: lib/fixup.rb [hunk 1]
```diff
@@ -154,6 +154,7 @@ def save_essential_deps(json_object)
   { pkg_name: 'ilmbase', pkg_rename: nil, pkg_deprecated: true, comments: 'Included in openexr.' },
   { pkg_name: 'imagemagick6', pkg_rename: nil, pkg_deprecated: true, comments: nil },
   { pkg_name: 'imagemagick', pkg_rename: nil, pkg_deprecated: true, comments: nil },
+  { pkg_name: 'itstool', pkg_rename: 'py3_itstool', pkg_deprecated: false, comments: 'Renamed to indicate switch to pip buildsyatem.' },
```
Comment (uberhacker) [inline lib/fixup.rb:157]: `buildsystem`?

File: packages/yelp_xsl.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Yelp_xsl < Autotools
      x86_64: '03e70d4b552cb539c1799f1709bf22c20ddcb281b508c462aa146db023f9801c'
   })
 
-  depends_on 'itstool'
+  depends_on 'py3_itstool'
```
Comment (uberhacker) [inline packages/yelp_xsl.rb:20]: Rename in `tools/packages.yaml` too?

Comment (satmandu) [reply packages/yelp_xsl.rb:20]: Will that mess up how your update checks script works?

Comment (uberhacker) [reply packages/yelp_xsl.rb:20]: Well, the script looks for the url but the name could be anything.

Comment (satmandu) [reply packages/yelp_xsl.rb:20]: Will adjust that then...

File: packages/metadata_cleaner.rb [hunk 3]
```diff
@@ -11,9 +11,9 @@ class Metadata_cleaner < Meson
 
   depends_on 'desktop_file_utils' # R
   depends_on 'gtk4' # R
-  depends_on 'itstool' # R
+  depends_on 'py3_itstool' # R
   depends_on 'libadwaita' # R
-  depends_on 'mesonbuild' => :build
+  depends_on 'py3_meson' => :build
```
Comment (uberhacker) [inline packages/metadata_cleaner.rb:16]: Fix in `tools/packages.yaml` also?

Comment (satmandu) [reply packages/metadata_cleaner.rb:16]: Done.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**